platform-operator/internal/deployer/kubernetes/kubernetes.go

```diff
@@ -264,10 +264,6 @@ func (d *KubernetesDeployer) createDeployment(ctx context.Context, component *pl
 			Name:  "CLIENT_NAME",
 			Value: clientId,
 		},
-		{
-			Name:  "CLIENT_ID",
-			Value: "spiffe://localtest.me/sa/" + component.Name,
-		},
```

Comment:
`CLIENT_ID` should not be provided as an env var. Instead, it's extracted from the mounted SVID.